packages/ui/fe-bundle/src/pages/[host]/[[...slug]].tsx

```diff
@@ -20,40 +21,40 @@ export declare namespace Docs {
     export interface Props {
         docs: FernRegistryDocsReadV2.LoadDocsForUrlResponse;
         resolvedUrlPath: ResolvedUrlPath;
-        typographyStyleSheet?: string;
+        typographyStyleSheet: string | undefined;
+        backgroundImageStyleSheet: string | undefined;
         nextPath: ResolvedUrlPath | null;
         previousPath: ResolvedUrlPath | null;
     }
 }
 
-function Typography({ stylesheet }: { stylesheet: string }) {
+function StyleSheets({ stylesheets }: { stylesheets: string[] }) {
```

Comment:
can we move this into a separate file?

Reply:
Yeah that's doable. I wanted to keep things simple for now but can move it.

Reply:
I tried moving this to a sincle instance, but Next JS will ignore the second rendered instance of the global style sheet. The result is to just concatenate all styles into a single string and inject the, globally.

If I were to use a regular style tag, Next, will inject them at the very top of the `<head/>`. This means they will be overriden by other global stylesheets in the head.

Reply:
@zachkirsch The solution was just to concatenate them to achieve these two outcomes:
- All configuration stylesheets should be serialized into a single stylesheet string.
- The final stylesheet string should be applied at the highest level of the CSS cascade for global styles. (i.e. it should be injected just before the `<head />` closing tag)

Not elegant, but it works. Will probably change this once we look into theming at a more concrete level.

Reply:
dope sg!